easybuild/easyconfigs/p/PyYAML/PyYAML-6.0.2-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'PyYAML'
+version = '6.0.2'
+
+homepage = 'https://github.com/yaml/pyyaml'
+description = "PyYAML is a YAML parser and emitter for the Python programming language."
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+sources = ['https://github.com/yaml/pyyaml/archive/refs/tags/%(version)s.tar.gz']
+checksums = ['9377c381ac3fccad8df73d96b5139ef8b1a2c57a0d913e95ab0a2275d66b5caa']
+
+builddependencies = [
+    ('binutils', '2.42'),
```

Comment:
`Cython` has been moved out of `Python-bundle-PyPI` in this toolchain generation.
i think we should add it here as a build dep.
maybe also set envvar PYYAML_FORCE_CYTHON=1 to enforce Cython?